hieradata/site/cp/role/rke.yaml

```diff
@@ -0,0 +1 @@
+profile::core::rke::version: "1.5.8"
```

Comment:
The `---` is missing.

Reply:
yeah, my fault, was added.

Reply:
Is this actually needed? It should get the default like the other sites.